### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by digitalsara** [ethto/animal-rescue-website#6]
On your home page: your logo isn't clickable (but it works well on all other pages)

For accessibility purposes, I suggest adding more skip links, especially when your home page skips only to the nav and main content (no focus state on your clickable cards from main, maybe add roles or focus states to those, too?)
